wdl/transforms/draft2/src/main/scala/wdl/transforms/draft2/wdlom2wom/WdlDraft2WomExecutableMaker.scala

```diff
@@ -11,5 +11,5 @@ object WdlDraft2WomExecutableMaker extends WomExecutableMaker[WdlNamespaceWithWo
     WdlDraft2InputParsing.buildWomExecutable(inputs.from.workflow, inputs.inputs)
   }
 
-  override def toWomExecutable(a: WdlNamespaceWithWorkflow, inputs: Option[String]): Checked[Executable] = toWomExecutable(ExecutableMakerInputs(a, inputs))
+  override def toWomExecutable(a: WdlNamespaceWithWorkflow, inputs: Option[String]): Checked[Executable] = toWomExecutable(ExecutableMakerInputs(a, List.empty, inputs))
```

Comment:
I do the same exact thing, switching back and forth, but can never decide what's more canonical: `List.empty` or `Nil`. I definitely use `Nil` in `case` statements, but for passing in as a variable, shrug...